android/build.gradle

```diff
@@ -1,5 +1,10 @@
 apply plugin: 'com.android.library'
 
+def DEFAULT_COMPILE_SDK_VERSION             = 23
```

Comment:
Could you add the other project properties as well? For `compileSdkVersion `, `buildToolsVersion` and `targetSdkVersion`